aiden-recommender/aiden_recommender/scrapers/scraper_aggregator.py

```diff
@@ -34,32 +61,39 @@ def _get_user_vector(profile_embedding_id: UUID) -> list[float]:
             raise RuntimeError("User has no vector embedding.")
         return user_vector
 
-    def search_jobs(self, search_query: str, location: str, profile_embedding_id: UUID, num_results: int = 15) -> list[str]:
+    async def search_jobs(self, search_query: str, location: str, profile_embedding_id: UUID, num_results: int = 15) -> list[str]:
```

Comment:
nit: func is returning job offers
```suggestion
    async def search_jobs(self, search_query: str, location: str, profile_embedding_id: UUID, num_results: int = 15) -> list[JobOffer]:
```